services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -139,16 +151,14 @@ const Asset = ({ libraryData }) => {
                 </dd>
               </Column>
               <Column className={clsx(dashboardStyles.subcolumn, dashboardStyles.subcolumnLinks)}>
-                <dt className={clsx(dashboardStyles.label)}>Demos</dt>
+                <dt className={clsx(dashboardStyles.label)}>Links</dt>
                 <dd className={dashboardStyles.meta}>
-                  <Link href={libraryPath}>
-                    <a className={dashboardStyles.metaLink}>Coming soon...</a>
-                  </Link>
+                  <ExternalLinks links={[assetData.content.demoLinks, externalDocsLink]} />
```

Comment:
If we're expecting demoLinks to be an array:
```suggestion
                  <ExternalLinks links={[...(get(assetData,'content.demoLinks', []), externalDocsLink]} />
```

Reply:
This is causing all kinds of weirdness when I try and update it, maybe we can pair up later? 🙏